CHANGELOG.md

```diff
@@ -51,6 +51,9 @@ with only minimum code changes required. See [the MD documentation](docs/bring_y
 - ([#439](https://github.com/microsoft/InnerEye-DeepLearning/pull/439)) Enable automatic job recovery from last recovery
   checkpoint in case of job pre-emption on AML. Give the possibility to the user to keep more than one recovery
   checkpoint.
+- ([#442](https://github.com/microsoft/InnerEye-DeepLearning/pull/442)) Enable custom scalar loss, prediction targets,
```

Comment:
This should mention the names of the parameters and functions added to the configs, and which config they have been added to (ModelConfigBase, ScalarConfig, etc) since there are quite a few additions.